src/backend/base/langflow/components/processing/dataframe_operations.py

```
@@ -150,36 +158,39 @@
-            return self.add_column(dataframe_copy)
-        if operation == "Select Columns":
-            return self.select_columns(dataframe_copy)
-        if operation == "Head":
-            return self.head(dataframe_copy)
-        if operation == "Tail":
-            return self.tail(dataframe_copy)
-        if operation == "Replace Value":
-            return self.replace_values(dataframe_copy)
-        msg = f"Unsupported operation: {operation}"
-
+        df = self.df.copy()
```

Ruff (0.11.9)

Avoid using the generic variable name `df` for DataFrames

(PD901)